Makefile

```diff
@@ -45,13 +52,26 @@ venvs/ot2: ot2monorepoClone
 	popd && \
 	deactivate
 
-# OVERRIDE_SETTINGS_DIR must be set to use opentrons v3
-# (otherwise it will try to access /data dir during 'import opentrons')
+.PHONY: parse-ot1
+parse-ot1: venvs/ot1 $(OT1_OUTPUT_FILES)
```

Comment:
Could we consider doing one of these two things?

1. Don't add `venvs/ot1` as a task dependency, because it's already a part of install and thus should be considered "setup"
2. Change it to an order-only-prerequisite


Also, see comment above about `.NOTPARALLEL` target